geopaparazzispatialitelibrary/src/eu/geopaparazzi/spatialite/database/spatial/core/daos/DaoSpatialite.java

```diff
@@ -436,9 +439,51 @@ public static void updateFeatureGeometry(String id, Geometry geometry, String ge
         sbIn.append(SpatialiteUtilities.SPATIALTABLE_ID_FIELD).append("=");
         sbIn.append(id);
         String insertQuery = sbIn.toString();
-        database.exec(insertQuery, null);
+        dbSpatialite.exec(insertQuery, null);
+    }
+    public static double[] longLat2Srid(Database dbSpatialite, double lon, double lat, String s_srid)  throws Exception {
```

Comment:
this is for the implementation of:

AbstractSpatialTable:
    /*
     * Convert a longitude and latitude to the table's
     * original SRID.
     */
    public abstract double[] longLat2Srid(double lon, double lat);

to show the present position on the screen
* which at the moment is only wsg84

The srid of the table being used would be sent to this function
* with the position returned in that srid
   * which could be placed then on the screen

It can also be used for any other position transformation.